cypress/integration/home_spec.js

```diff
@@ -13,7 +13,7 @@ describe('Home', () => {
     it('Should display a log in form', () => {
         cy.get('form').should('be.visible')
           .get('input').should('have.length', 2)
-          .get('input[name=userName]').should('have.attr', 'type', 'text')
+          .get('input[name=userName]').should('have.attr', 'type', 'email')
```

Comment:
I like this change, so it alerts the user if they didn't type in an email for the username